bin/test-documentation

```diff
@@ -7,25 +7,16 @@
 
 runSbt unidoc
 
-EXTRA_TASK=""
-
-# Check if it is a scheduled build
+declare -a EXTRA_TASKS
```

Comment:
`-a` means "array", so I'm concerned doing this without the other changes (adding with `+=` and expanding with `[@]`) might affect things.

Reply:
Done.

Good catch, I was too fast when changing things here.